### ISSUE TITLE
websockets not working local

### ISSUE THREAD

**Issue by wingmanAT** [jaggedsoft/php-binance-api#112]
Hi,
I've installed your API on my local machine. I've installed the newest version of XAMPP  with PHP 7.2.4.
I've installed composer through the installer-file, selected the right file for php.exe.

Xampp is working, I build a script and I am able to access data through the rest-api. 
But nothing works with the websockets.api, even the example-files are not working.

Here are the errors I get:

From the web_socket_chart.php (example-file) I got this error:
`Fatal error: Uncaught Error: Class 'React\EventLoop\Factory' not found in E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\php-binance-api.php:1200 Stack trace: #0 E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\examples\web_socket_chart.php(13): Binance\API->chart(Array, '15m', Object(Closure)) #1 {main} thrown in E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\php-binance-api.php on line 1200`

If I try the web_socket_depth.php I get this:
`Fatal error: Uncaught Error: Class 'React\EventLoop\Factory' not found in E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\php-binance-api.php:1106 Stack trace: #0 E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\examples\web_socket_depth.php(18): Binance\API->depthCache(Array, Object(Closure)) #1 {main} thrown in E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\php-binance-api.php on line 1106`

I can't access any of the websocket-data. Maybe you can give me a hint?

Thank you.

**Comment by dmzoneill** [comment]
What is the output of 
```
composer -vvv install
```
In the same folder as composer.json

**Comment by dmzoneill** [comment]
Did you follow the instructions to create the token, to get passed the rate limit and download the remaining dependancies?

**Comment by wingmanAT** [comment]
ok. i deinstalled everything and installed again.
I have this folder structure:
here is my project:
E:\xampp7\htdocs\binance\
I navigate to this folder inside of the console and called 

> composer require "jaggedsoft/php-binance-api @dev"
he creates a composer.json file and many subfolders:

E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\

inside of the last folder there is also a composer.json. is this correct?
I run this file 
http://localhost/binance/vendor/jaggedsoft/php-binance-api/examples/web_socket_chart.php

and he says:

> Warning: require(php-binance-api.php): failed to open stream: No such file or directory in E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\examples\web_socket_chart.php on line 3
> 
> Fatal error: require(): Failed opening required 'php-binance-api.php' (include_path='E:\xampp7\php\PEAR') in E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\examples\web_socket_chart.php on line 3
> 

why is he trying to include from php\PEAR? ... Inside of the installation process of composer he asked me where the php.exe is. This file is inside of E:\xampp7\php\ ... is this the "problem"?...

if I call inside of my project-folder: E:\xampp7\htdocs\binance\
> composer -vvv install

he says this:

> E:\xampp7\htdocs\binance>composer -vvv install
Reading ./composer.json
Loading config file ./composer.json
Checked CA file E:\xampp7\apache\bin\curl-ca-bundle.crt: valid
Executing command (E:\xampp7\htdocs\binance): git branch --no-color --no-abbrev -v
Executing command (E:\xampp7\htdocs\binance): git describe --exact-match --tags
Executing command (E:\xampp7\htdocs\binance): git log --pretty="%H" -n1 HEAD
Executing command (E:\xampp7\htdocs\binance): hg branch
Executing command (E:\xampp7\htdocs\binance): fossil branch list
Executing command (E:\xampp7\htdocs\binance): fossil tag list
Executing command (E:\xampp7\htdocs\binance): svn info --xml
Failed to initialize global composer: Composer could not find the config file: C:/Users/human/AppData/Roaming/Composer/composer.json
To initialize a project, please create a composer.json file as described in the https://getcomposer.org/ "Getting Started" section
Reading E:\xampp7\htdocs\binance/vendor/composer/installed.json
Running 1.6.4 (2018-04-13 12:04:24) with PHP 7.2.3 on Windows NT / 10.0
Reading ./composer.lock
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Reading ./composer.lock
Resolving dependencies through SAT
Dependency resolution completed in 0.001 seconds
Analyzed 74 packages to resolve dependencies
Analyzed 150 rules to resolve dependencies
Nothing to install or update
Generating autoload files

so everything is fine, but he is doing something weird...

**Comment by wingmanAT** [comment]
EDIT: I got it done.

I copied all files outside of E:\xampp7\htdocs\binance\vendor\jaggedsoft\php-binance-api\

directly to my folder:
E:\xampp7\htdocs\binance\

even the examples have false link structre inside. If I copy them inside of my main folder, they are working.
Now he is able to pull files from web_socket_chart.php

oh guys.... this is complicated for no reason. Just put the files online and make them downloadable. All this installation process with false paths is not needed. But good work indeed! Thank you for helping!
